### PR TITLE
Kk/update readme

### PR DESCRIPTION
**Pull request by karlkallas**
New readme that improves developer experience by:

1. fixing broken links, adding useful links
2. removing unnecessary parts
3. improve quickstart structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,9 @@
 </p>
 
 
-## Overview
+## 🕊️ OpenCopilot Overview
 
-Copilots are becoming the new paradigm how to build successful LLM-based applications, as seen by Github , Shopify, Brex, Hubspot, etc Copilots. Yet, building a Copilot that goes beyond a Twitter demo is extremely complex as it's time-consuming, unreliable and feels like a massive undertaking. Moreover, existing solutions such as Microsoft Copilot Stack are closed-source. Building an LLM-app today feels like:
+Copilots are becoming the new paradigm how to build successful LLM-based applications, as seen by [Github](https://github.com/features/copilot), [Shopify](https://www.shopify.com/magic), [Brex](https://www.brex.com/journal/press/brex-openai-ai-tools-for-finance-teams), [Hubspot](https://app.hubspot.com/chatspot/chat), etc Copilots. Yet, building a Copilot that goes beyond a Twitter demo is extremely complex as it's time-consuming, unreliable and feels like a massive undertaking. Moreover, existing solutions such as Microsoft Copilot Stack are closed-source. Building an LLM-app today feels like:
```
Comment (taivop) [inline README.md:37]: Now we're duplicating links here and in section below - is the below section even necessary?

File: README.md [hunk 2]
```diff
@@ -105,59 +110,38 @@ Parrot Copilot answer in Markdown:
...
-If you'd like to have a front-end for your Copilot, then you can easily deploy it. As a pre-requisite, you need to have [`pnpm`](https://pnpm.io/) installed.
-
-First, clone the [opencopilotdev/opencopilot-frontend](https://github.com/opencopilotdev/opencopilot-frontend) repository:
-
-```bash
-git clone https://github.com/opencopilotdev/opencopilot-frontend
-```
-
-Then, setup the environment variables:
+If you'd like to set up an out of the box front-end for your Copilot, follow the instructions in the [opencopilotdev/opencopilot-frontend](https://github.com/opencopilotdev/opencopilot-frontend) repository.
```
Comment (taivop) [inline README.md:140]: I like the approach of sending person to the frontend repo then.

File: README.md (hunk 1 above)
Comment (karlkallas) [reply README.md:37]: showing categories (use cases in general) helps, but it feels like a duplicate for sure, can remove links from the beginning now and propose a minified readme in another PR, wdyt?

Comment (taivop) [reply README.md:37]: I was actually thinking of having links only at the top, but sure we can go your route - especially if anyway gonna remake this soon.

**Review by taivop — APPROVED** [review]

**Pull request merged**